python/kwiver/vital/types/sfm_constraints.cxx

```diff
@@ -27,29 +29,13 @@ PYBIND11_MODULE( sfm_constraints, m )
                   &kv::sfm_constraints::get_local_geo_cs,
                   &kv::sfm_constraints::set_local_geo_cs )
   .def( "get_camera_position_prior_local",
-        ( bool ( kv::sfm_constraints::* ) ( kv::frame_id_t, kv::vector_3d& ) const )
-        &kv::sfm_constraints::get_camera_position_prior_local )
-  .def( "get_camera_position_prior_local",
-        ( bool ( kv::sfm_constraints::* ) ( kv::frame_id_t, kv::rotation_d& ) const )
```

Comment:
@johnwparent, so, I'm not sure how this ever actually compiled, as there was not such an overload of `get_camera_position_prior_local` (and indeed, as you noticed, `get_camera_position_prior_local` is not even overloaded). The binding for `get_camera_orientation_prior_local` however was missing; I've fixed that in the most recent version; PTAL.

Reply:
I also have no idea how that compiled, especially considering that it didn't in my local container... Oh shoot, I missed that one, sorry. Your implementation looks good however.